ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/dualstack/provision/cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - ref: ovn-conf 
```

Comment:
seem like `ovn-conf-baremetal-dualstack` also set `OVNKubernetes` networking type, so I guess we can remove `ovn-conf` step here.

Reply:
`ovn-conf` sets the `GATEWAY_MODE` as well, so we might still need it.

Reply:
I guess all the OVN steps need cleanup.

Reply:
> ovn-conf sets the GATEWAY_MODE as well, so we might still need it.

Yeah, saw that, but seem like this workflow did not set GATEWAY_MODE.